Spanner/tests/System/PgReadTest.php

```diff
@@ -80,8 +85,8 @@ public function testRangeReadSingleKeyOpen()
 
         $res = $db->read(self::$rangeTableName, $keyset, array_keys(self::$dataset[0]));
         $rows = iterator_to_array($res->rows());
-        $this->assertNotContains(self::$dataset[0], $rows);
-        $this->assertNotContains(self::$dataset[10], $rows);
+        $this->assertStrngNotContainsString(self::$dataset[0], $rows);
```

Comment:
```suggestion
        $this->assertStringNotContainsString(self::$dataset[0], $rows);
```

Reply:
There are a handful of other instances of this through the file.

Reply:
A little concerning that it didn't throw an error...

Reply:
Nevermind, these are for the system tests, which I didn't run. I'll run them locally to make sure they're alright.